source/manual/alerts/app-healthcheck-failed.html.md

```diff
@@ -15,3 +15,9 @@ endpoint doesn't respond successfully, it triggers this alert.
 
 To solve this issue, look at the logs of the application to see what
 is wrong.
+
+## Application healthcheck documentation
+
+- [Search API Healtcheck failed](search-api-healthcheck)
```

Comment:
These links are wrong, it should be `[title][anchor]`

Reply:
Thanks a lot, fixed.